label/subword_nmt/apply.py

```diff
@@ -0,0 +1,144 @@
+__all__ = ["ApplyBPEModelToLexiconJob", "ApplyBPEToTextJob"]
```

Comment:
these jobs probably don't work for sentencepiece models, right?

Will there be apply Jobs for the new module as well?

Reply:
Currently they are not needed, but we can add them once they are. I can add a "TODO" issue for that if you want.